physionet-django/templates/about/software_content_test.html

```diff
@@ -0,0 +1,15 @@
+<section id="overview">
+    <h2>Overview</h2>
+  <p>This page displays an alphabetical list of all software projects on PhysioNet. To search content on PhysioNet, <a href="{% url 'content_index' %}">visit the search page</a>. Enter the search terms, add a filter for resource type if needed, and select how you would like the results to be ordered (for example, by relevance, by date, or by title).</p>
+  <p>Many of the time series datasets on PhysioNet (for example, ECGs) are shared in waveform-database (WFDB) compatible file formats.  For more information on these formats, see the <a href="https://www.physionet.org/physiotools/wag" />WFDB Applications Guide</a>. WFDB-formatted datasets can also be viewed online with <a href="{% url 'lightwave_home' %}">LightWAVE</a>.</p>
```

Comment:
I'm not sure that the hardcoded URL ("https://www.physionet.org/physiotools/wag") is what we want here, but let's leave it for now.

Reply:
How do I get rid of it? I couldn't find an existing URL path set up for it unless I wasn't looking right?

Reply:
Not sure from the top of my head. I think we get the domain `https://www.physionet.org/` elsewhere, which might be a start (or if not this, then a relative URL might be more flexible).